### ISSUE TITLE
OffsetDateTimeMapperTest is failing

### ISSUE THREAD

**Issue by patrox** [dropwizard/dropwizard#1505]
While testing something completely unrelated, I stumbled upon two failed tests, which IMO seem to fail because of DST offset, as:

```
  OffsetDateTimeMapperTest.mapColumnByIndex:44 expected:<...12-03T10:15:30.375+0[2]:00> but was:<...12-03T10:15:30.375+0[1]:00>
  OffsetDateTimeMapperTest.mapColumnByName:25 expected:<...12-03T10:15:30.375+0[2]:00> but was:<...12-03T10:15:30.375+0[1]:00>
```

In the above output you can see that the [`actual`](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/args/OffsetDateTimeMapperTest.java#L22-L22) value:

> 2007-12-03T10:15:30.375+01:00 // doesn't include DST offset

is **NOT** equal to the [`expected`](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/args/OffsetDateTimeMapperTest.java#L25-L25) value:

> 2007-12-03T10:15:30.375+02:00 // includes DST offset

Should I take care of that ?

**Comment by patrox** [comment]
the same applies to [`OffsetDateTimeArgumentTest.apply`](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/args/OffsetDateTimeArgumentTest.java#L20-L20)

**Comment by jplock** [comment]
Yes please! That'd be great. Thanks

**Comment by patrox** [comment]
@jplock, @joschi could you please have a look at the above PR addressing the described issue ?

**Comment by patrox** [comment]
Addressed in #1506
